contribs/freight/src/main/java/org/matsim/contrib/freight/utils/FreightUtils.java

```diff
@@ -209,6 +209,20 @@ public static CarrierVehicleTypes getCarrierVehicleTypes(Scenario scenario) {
 		}
 		return types;
 	}
+	
+	/** Adds the CarrierVehicleTypes to the scenario.
+	 * @param scenario
+	 * @param newTypes
+	 */
+	public static void addCarrierVehicleTypes(Scenario scenario, CarrierVehicleTypes newTypes) {
```

Comment:
I would like to have a short test for this. to see, if the types are added correctly